src/index.html

```diff
@@ -3,11 +3,261 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Miami</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <title>The MET</title>
+    <link
+      rel="shortcut icon"
+      href="./images/photos/THEMET.png"
+      type="image/png">
+    <!-- <link rel="stylesheet" href="./styles/main.scss"> -->
+    <link rel="stylesheet" href="/dist/css/main.css">
     <script src="scripts/main.js" defer></script>
   </head>
   <body>
-    <h1>Miami</h1>
+    <h1 hidden>The MET</h1>
+    <!-- <div class="container"> -->
```

Comment:
continuing our discussion - asking `do you need this wrapper?` did not mean it's not needed.
When you don't have - how do you shape all your contents to follow the adaptive design?
so you need it to shape all the children elements.
Now you have a horizontal scroll to resolve.
![Snapshot 2022-04-13 11 31 06](https://user-images.githubusercontent.com/39308717/163134928-c4b6e87b-6a2d-43be-bb72-0996f62cda93.png)